### PR TITLE
Improve release test

### PR DESCRIPTION
**Pull request by hanssv**
It wasn't checking that all nodes had reached the stipulated height. It sometimes got stuck waiting for a ping to do a retry (2 minutes is unnecessarily long). And it hit a Python snag so all communication was to the same node since the configuration was copied by reference...

### CODE REVIEW

File: py/requirements-win32.txt [hunk 1]
```diff
@@ -2,5 +2,6 @@
 ## This is a subset of the dependencies defined in py/requirements.txt. These dependencies can't be installed via the OS
 ## package manager.
 ##
+urllib3>=1.24.1
```
Comment (lucafavatella) [inline py/requirements-win32.txt]: Maybe it shall be bumped up in both requirements.txt and this one?

Comment (hanssv) [reply py/requirements-win32.txt]: I honestly have no idea, this is just a stab in the wild. I have very little insight in Python deps...

File: scripts/windows/msys2_prepare.bat [hunk 2]
```diff
@@ -128,6 +128,9 @@ IF EXIST "%WIN_STYRENE_PATH%" IF "%FORCE_STYRENE_REINSTALL%" NEQ "true" GOTO STY
 %BASH_BIN% -lc "cd \"${ORIGINAL_TEMP}/styrene\" && %PIP% install ."
 :STYRENEINSTALLED
 
+@echo Patching request
+%BASH_BIN% -lc "sed -i -e 's/assert minor <= 24/assert minor <= 25/' C:/msys64/mingw64/lib/python3.7/site-packages/requests/__init__.py"
```
Comment (dincho) [inline scripts/windows/msys2_prepare.bat]: 🙀

Comment (hanssv) [reply scripts/windows/msys2_prepare.bat]: Move along nothing to see, don't worry I'm a professional ;-)

**Comment by hanssv** [comment]
Haha, `pacman` included the new version of `request` package for Python today... Will remove the unnecessary `sed` bit before we merge this...

File: scripts/windows/msys2_prepare.bat [hunk 3]
```diff
@@ -98,6 +98,9 @@ rem Upgrade the MSYS2 platform
 rem Install required tools
 %BASH_BIN% -lc "%PACMAN% %PACMAN_PACKAGES% %PACMAN_PYTHON_PACKAGES%"
 
+@echo Patching request
+%BASH_BIN% -lc "sed -i -e 's/assert minor <= 24/assert minor <= 25/' C:/msys64/mingw64/lib/python3.7/site-packages/requests/__init__.py"
```
[the remark this answers is not in the document]
Comment (hanssv) [reply scripts/windows/msys2_prepare.bat]: No sed in the last version?!

**Pull request merged**